doc/api/crypto.md

```diff
@@ -972,6 +1130,9 @@ The `key` is the raw key used by the `algorithm` and `iv` is an
 [buffers][`Buffer`].
 
 ### crypto.createDiffieHellman(prime[, prime_encoding][, generator][, generator_encoding])
```

Comment:
I used v0.5.0 here as per this [comment](https://github.com/nodejs/node/pull/8206#discussion_r75768811) but in the same [pr](https://github.com/nodejs/node/pull/8196) I used two different versions ([v0.11.13](https://github.com/nodejs/node/pull/8196/files#diff-3fc1b3cd67f8ff2b48a0fb33b30de403R432) and [v0.1.99](https://github.com/nodejs/node/pull/8196/files#diff-3fc1b3cd67f8ff2b48a0fb33b30de403R457)) for `dgram.createSocket`.
I'm going to do the same here.